client/src/pages/SymptomsCheck.js

```diff
@@ -248,7 +262,7 @@ const Welcome = (props) => {
     </WizardStep>
   );
 };
-
+//all steps require refactor
```

Comment:
not sure what the purpose if this comment is, as a general rule I think it's best to avoid comments that don't really add value and aren't strictly necessary (prefer self-describing code that doesn't need explanation if possible)

Reply:
Yea, sorry about that. This is more of a reminder for enhancement in the future. Removed!